test/cli/debug.t

```diff
@@ -260,9 +243,7 @@
               before: (* before x binding #2 *)
               Pexp_let Nonrec
               [
-                <def> (a.ml[5,90+2]..[10,210+28])
-                comments
-                   after: (* after unit *)
+                <def>
```

Comment:
The comment is not preserved.

Reply:
So the `(* after unit *)` comment is not attached anywhere?

Reply:
My previous comment is not exact. This node has a location and the comment is still attached to it. Printast was not printing it.